documentation/host/how-to-self-test.mdx

```diff
@@ -0,0 +1,179 @@
+---
+
+<script type="application/ld+json" dangerouslySetInnerHTML={{
+  __html: JSON.stringify({
+    "@context": "https://schema.org",
+    "@type": "FAQPage",
+    "mainEntity": [
+      {
+        "@type": "Question",
+        "name": "What does meeting minimum requirements mean for verification?",
+        "acceptedAnswer": {
+          "@type": "Answer",
+          "text": "Meeting these minimum requirements makes your machine eligible for verification, but does not automatically guarantee verification. The requirements ensure your machine has the basic quality and reliability benchmarks needed to run customer workloads effectively."
+        }
+      },
+      {
+        "@type": "Question",
+        "name": "What components does the self-test check?",
+        "acceptedAnswer": {
+          "@type": "Answer",
+          "text": "The self-test verifies driver setup including CUDA configuration, network speed and stability, open ports and connectivity, PCIe bandwidth, GPU VRAM capacity and reliability, system RAM and CPU performance, and overall machine reliability under simulated workloads."
+        }
+      },
+      {
+        "@type": "Question",
+        "name": "How do I install the Vast CLI?",
+        "acceptedAnswer": {
+          "@type": "Answer",
+          "text": "Follow the official setup guide to install the Vast CLI. URL:https://docs.vast.ai/cli/get-started"
+        }
+      },
+      {
+        "@type": "Question",
+        "name": "How do I set up my API key?",
+        "acceptedAnswer": {
+          "@type": "Answer",
+          "text": "Get your API key from the web console Keys page and authenticate your vastCLI with your Vast.ai account using the vastai set api-key command."
+        }
+      },
+      {
+        "@type": "Question",
+        "name": "How do I run the self-test?",
+        "acceptedAnswer": {
+          "@type": "Answer",
+          "text": "Execute the self-test command on your listed machine when no clients are currently renting it to verify performance and configuration standards. Use the command: vastai self-test machine <machine_id>"
+        }
+      },
+      {
+        "@type": "Question",
+        "name": "What should I do if the self-test fails?",
+        "acceptedAnswer": {
+          "@type": "Answer",
+          "text": "If the test fails, the CLI will display detailed reasons for failure. Apply the fixes and rerun the test. You can also use the --ignore-requirements flag to check rentability in a relaxed mode."
+        }
+      },
+      {
+        "@type": "Question",
+        "name": "What does the --ignore-requirements flag do?",
+        "acceptedAnswer": {
+          "@type": "Answer",
+          "text": "The --ignore-requirements flag runs the test in a relaxed mode, bypassing some checks. Even if the test passes in this mode, your machine does not meet the minimum verification requirements. Your machine must still have at least three direct open ports."
```

Comment:
The --ignore-requirements flag runs the test in a relaxed mode, **bypassing the verification benchmarks**. Even if the test passes in this mode, your machine does not **necessarily** meet the minimum verification requirements. Your machine must still have at least three direct open ports **to run these tests**.